src/session/types.ts

```diff
@@ -27,3 +27,14 @@ export interface ISignInInfo {
   signedIn?: boolean;
   userProfile?: IUserProfile;
 }
+
+export interface IPromiseToFileSession {
+  company_in_context: string;
+  ptf_requests: IPromiseToFileRequest[];
+}
+
+export interface IPromiseToFileRequest {
+  company_number: string;
+  extension_request_id: string;
```

Comment:
Guessing this shouldn't be named 'extension_request_id' but 'promise_to_file_request_id'?

Reply:
changed